scripts/miniapps/CoMD.md

```diff
@@ -0,0 +1,110 @@
+# Instruction for building and running CoMD
+
+Implementation: C, MPI+OpenMP
+
+Useful links:
+- ExMatEx CoMD website: http://www.exmatex.org/comd.html
+- Build instructions: http://exmatex.github.io/CoMD/doxygen-mpi/pg_building_comd.html
+- GitHub repository: https://github.com/exmatex/CoMD
+
+## How to build
+
+1. Download from the GitHub repository
+   ```
+% git clone https://github.com/exmatex/CoMD
+   ```
+
+   There is a top-level README.md with a somewhat useful link in it.
+
+2. Load necessary PrgEnv module, set PATH and LD_LIBRARY_PATH and set
+recommended environment variables, e.g. for OMPI, etc.
+
+3. Create and modify Makefile in the `src-openmp` directory:
+   ```
```

Comment:
I changed this to be quoted (`Makefile`) so that it was clear this was a filename.